src/main/java/ailurus/Parser.java

```diff
@@ -0,0 +1,95 @@
+package ailurus;
+
+import ailurus.task.*;
```

Comment:
I think it would be better to import the classes in the task package explicitly for better documentation.